src/csharp/Intel/Iced.UnitTests/Intel/AssemblerTests/AssemblerTests64.cs

```diff
@@ -97,6 +97,23 @@ public sealed partial class AssemblerTests64 {
 		}
 #endif
 
+
+		[Fact]
+		public void TestDeclareData_db_array() {
+			TestAssemblerDeclareData(c => c.db(Array.Empty<byte>()), Array.Empty<byte>());
+			TestAssemblerDeclareData(c => c.db(new byte[] { 1 }), new byte[] { 1 });
+			TestAssemblerDeclareData(c => c.db(new byte[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16 }), new byte[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16 });
+			TestAssemblerDeclareData(c => c.db(new byte[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17 }), new byte[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17 });
+			TestAssemblerDeclareData(c => c.db(new byte[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20 }), new byte[] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20 });
```

Comment:
This one is still 20 elems instead of 32 elems. This test should have 32 elems and the next line should have 33 elems.